src/main/resources/META-INF/plugin.xml

```diff
@@ -118,10 +118,6 @@
                        groupName="Shopware"
                        enabledByDefault="true" level="WARNING"
                        implementationClass="de.espend.idea.shopware.inspection.ShopwareBoostrapInspection"/>
-      <localInspection groupPath="Shopware" shortName="ShopwareBoostrapCheckLicense" displayName="BoostrapCheckLicense"
```

Comment:
maybe remove also the class itself?

Reply:
I have removed it, the PR contains two files 😊

Reply:
ah sry sure ;)